cai_causal_graph/time_series_causal_graph.py

```diff
@@ -208,7 +185,7 @@ def get_stationary_graph(self) -> TimeSeriesCausalGraph:
         neg_lag, pos_lag = lags[0], lags[-1]
 
         # now extend the minimal graph to the current max and min lag to match the current graph
-        self._stationary_graph = minimal_graph.extend_graph(-neg_lag, pos_lag)
+        self._stationary_graph = minimal_graph.extend_graph(-neg_lag, pos_lag, include_all_parents=False)
```

Comment:
Was the plan to change the default to `True` for `include_all_parents` or did MLE/Dara land on a different solution? I saw no one replied to your question on team_dara.

Reply:
I am still waiting for them. Claudia said she is fine with it, but I said I'll wait for her to give me a formal reply. I figured it can be done as a separate PR (before we release a new minor).

Reply:
Sounds good. Just noticed this was set explicitly so it was hinting that default was changing. If we do change it then the docs actually need an update as they give some examples of how extend works.